torch_xla/csrc/ops/native_dropout.cpp

```diff
@@ -0,0 +1,36 @@
+#include "torch_xla/csrc/ops/native_dropout.h"
+
+#include "torch_xla/csrc/helpers.h"
+#include "torch_xla/csrc/lowering_context.h"
+#include "torch_xla/csrc/xla_lower_util.h"
+
+namespace torch_xla {
+namespace {
+
+xla::Shape NodeOutputShape(const torch::lazy::Value& input) {
+  xla::Shape input_shape = GetXlaShape(input);
+  return xla::ShapeUtil::MakeTupleShape({input_shape, input_shape});
```

Comment:
`xla::Shape` contains dtype info. In PyTorch, there is Bool Tensor in PyTorch, but there is no `bool` in `xla::PrimitiveType`. So let's use `uint8` instead.

```
shape.set_element_type(xla::PrimitiveType::U8);
```

Reply:
In the test, we may check the output tensor type, note that we may hit dtype mismatch `bool` vs `uint8`. If that happens, let's just assert `uint8` for the mask output and add a comment in the test code.

Reply:
I think bool in XLA is called `xla::PRED`

Reply:
Thanks, we have converted the mask tensor into `xla::PRED`.